### ISSUE TITLE
"Copy Short URL" -> "Copy Original URL" ?

### ISSUE THREAD

**Issue by MadtownLems** [markjaquith/page-links-to#122]
I was a bit confused by the "Copy Short URL" that shows up when you hover over a post using Page Links To in the dashboard. Am I missing something, or would this be better off being labeled: "Copy Original URL"?

I expect a "short url" to be something like: x.yz/abc123 , and at least for us, the "short url" is actually longer than the redirected to URL approximately half the time.

**Comment by markjaquith** [comment]
Yeah. It depends on how long your domain is and how long the thing you're linking to is. I guess the main point is that this is a URL on your site... so that you can, say, include it in an email or a social campaign and it looks more "on brand." Maybe "Original URL" is the way to go. I'll give it some more thought!

**Comment by MadtownLems** [comment]
Awesome. Just for some more examples of how people are using it, a lot of our users don't just use it to link off to another site. They use it to link a post to an instance of a custom post type that isn't included in the standard loop, for example.  Perhaps a post such as "Just announced: the 2020 Whatever Symposium!" and have that post show up on the blog and in feeds, but following the link takes you straight to an Event. 
I believe this is further argument for something like "Original URL" as opposed to something that would be like "Local URL", "Branded URL" or "On-site URL"
